barretenberg/cpp/pil/vm2/sha256.pil

```diff
@@ -0,0 +1,375 @@
+include "precomputed.pil";
+namespace sha256;
+
+    // Memory ops
+    pol commit state_offset;
```

Comment:
We are not indenting in the other files, since there is only 1 namespace expected, does it make sense?

Reply:
i like the indentation - i feel like it makes things easier to read tbh